src/services/autoMailer.js

```diff
@@ -0,0 +1,31 @@
+import sgMail from '@sendgrid/mail';
+import dotenv from 'dotenv';
+import emailTemplatesFunction from '../utils';
+
+dotenv.config();
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const sendEmail = async (receiver, category, data) => {
```

Comment:
What is `receiver, category, data`. You need to add appropriate documentation to help us under what this function does and what the parameters are.

Reply:
@eokwukwe , since receiving notification about your comment, all my effort to get electricity has been fruitless, though I have not giving up, I'm assured of having a generator fixed first thing tomorrow morning. I had to switch to a desktop computer as my laptop's display is still unstable at this time

Reply:
Where is @uthdev?